src/components/ShippingDetails/ShippingDetails.js

```diff
@@ -1,22 +1,42 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Shipping from '../Icons/Shipping';
```

Comment:
Separate external and internal imports with new line and fix import order as per README.